docs/unified/novacustom/hcl.md

```diff
@@ -130,6 +130,17 @@ issue from which the report is sourced.
         1. Rev.C1 and Rev.E
         2. DP Alt mode: Synaptics VMM5310 DP MST hub, two upstream DP 1.4 lanes,
            DSC 1.2 decompression, up to 2x 4K60 + 1x 4K30 depending on source.
+    === "V540TU MTL Ultra 5 14th Gen."
```

Comment:
```suggestion
    === "V54 / V56 14th Gen."
```

All MTL models (dGPU/iGPU, 14 and 16 inch) will be tested with that dock

Reply:
ok, other than that the syntax is correct? I believe i have no way to preview it, i might be wrong tho.

Reply:
Syntax looks good. In the future if you want to preview locally you can follow https://github.com/Dasharo/docs?tab=readme-ov-file#local-build

Reply:
thanks, i will use that in the future for sure